runtime/doc/news.txt

```diff
@@ -125,6 +125,14 @@ The following changes may require adaptations in user config or plugins.
 • Returning any truthy value from a callback passed to |nvim_create_autocmd()|
   (rather than just `true`) will delete the autocommand.
 
+• |vim.lsp.util.extract_completion_items()| will no longer return reliable
+  results, since it does not apply `itemDefaults` when its input is a
+  `CompletionList`.
+  Moreover, since support for LSP `completionList.itemDefaults` was added,
+  some third party plugins might be negatively impacted because of lack of
+  support by language servers. If necessary, the respective capability can be
```

Comment:
Small correction: They're negatively affected if the servers support it but the plugin doesn't.